src/GitHub.App/ViewModels/LoginTabViewModel.cs

```diff
@@ -18,6 +19,7 @@
 
 namespace GitHub.ViewModels
 {
+    [SuppressMessage("Microsoft.Design", "CA1001:TypesThatOwnDisposableFieldsShouldBeDisposable")]
```

Comment:
`RecoveryCommand` leaks a disposable for the `canExecute`, but it should be disposed automatically when the viewmodel gets disposed of (which should happen when the windows is closed), so it should be fine.